geometry_msgs/Point[] facet_mids
geometry_msgs/Point[] normals